app/src/utils/nfcScanner.ts

```diff
@@ -50,33 +50,6 @@ export const parseScanResponse = (response: unknown) => {
     : handleResponseIOS(response);
```

Comment:
nit: feel like we could also move additional or all the methods from this file over to the sdk. other devs would probably find them useful

Reply:
leaving the code in the app as-is as discussed